app/src/main/java/com/vishnu/whatsappcleaner/ui/theme/Theme.kt

```diff
@@ -134,25 +134,26 @@ fun WhatsAppCleanerTheme(
     }
 
     val typography = Typography(
-        displayLarge = Typography.displayLarge.copy(fontFamily = fontFamily),
-        displayMedium = Typography.displayMedium.copy(fontFamily = fontFamily),
-        displaySmall = Typography.displaySmall.copy(fontFamily = fontFamily),
+        displayLarge = Typography.displaySmall.copy(fontFamily = fontFamily),
+        displayMedium = Typography.headlineLarge.copy(fontFamily = fontFamily),
+        displaySmall = Typography.headlineMedium.copy(fontFamily = fontFamily),
 
-        headlineLarge = Typography.headlineLarge.copy(fontFamily = fontFamily),
-        headlineMedium = Typography.headlineMedium.copy(fontFamily = fontFamily),
-        headlineSmall = Typography.headlineSmall.copy(fontFamily = fontFamily),
+        headlineLarge = Typography.headlineSmall.copy(fontFamily = fontFamily),
+        headlineMedium = Typography.titleLarge.copy(fontFamily = fontFamily),
+        headlineSmall = Typography.titleMedium.copy(fontFamily = fontFamily),
 
-        titleLarge = Typography.titleLarge.copy(fontFamily = fontFamily),
-        titleMedium = Typography.titleMedium.copy(fontFamily = fontFamily),
-        titleSmall = Typography.titleSmall.copy(fontFamily = fontFamily),
+        titleLarge = Typography.titleSmall.copy(fontFamily = fontFamily),
+        titleMedium = Typography.bodyLarge.copy(fontFamily = fontFamily),
+        titleSmall = Typography.bodyMedium.copy(fontFamily = fontFamily),
 
-        bodyLarge = Typography.bodyLarge.copy(fontFamily = fontFamily),
-        bodyMedium = Typography.bodyMedium.copy(fontFamily = fontFamily),
-        bodySmall = Typography.bodySmall.copy(fontFamily = fontFamily),
+        bodyLarge = Typography.bodySmall.copy(fontFamily = fontFamily),
+        bodyMedium = Typography.labelLarge.copy(fontFamily = fontFamily),
+        bodySmall = Typography.labelMedium.copy(fontFamily = fontFamily),
+
+        labelLarge = Typography.labelSmall.copy(fontFamily = fontFamily),
+        labelMedium = Typography.labelSmall.copy(fontFamily = fontFamily),
+        labelSmall = Typography.labelSmall.copy(fontFamily = fontFamily) // Already smallest, no change
 
-        labelLarge = Typography.labelLarge.copy(fontFamily = fontFamily),
-        labelMedium = Typography.labelMedium.copy(fontFamily = fontFamily),
-        labelSmall = Typography.labelSmall.copy(fontFamily = fontFamily)
     )
```

Comment:
please don't change this. this was added to change the font globally. ([ref](https://stackoverflow.com/a/76383549/9652621)). the global font sizes are defined on `Type.kt`, FYI.